sx-interaction.el

```diff
@@ -278,7 +278,12 @@ likes."
        (sx-question-get-from-comment .site_par .comment_id) 'focus)
       (sx--find-in-buffer 'comment .comment_id))
      (.notification_type
-      (sx-message "Viewing notifications is not yet implemented"))
+;     (sx-message "Viewing notifications is not yet implemented"))
```

Comment:
We can probably just delete this line.

Reply:
I left it in because what I did wasn't really a solution for the lack of implementation, but rather a stop-gap to give the user the ability to continue using stackexchange in another interface (the web browser). Whenever sx.el does implement support for those type of pages, it would be appropriate to remove my code snippet, but for now a user has no way to perform the tasks associated with the notification, eg. review someone's suggested edits.